src/app/error-page/error-page.routing.module.ts

```diff
@@ -0,0 +1,24 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule,Routes } from '@angular/router'
+
+import { ErrorPageComponent } from './error-page.component';
+
+
+
```

Comment:
Please follow “Single Responsibly Principle” One entity per file. Move this routing to separate file

Reply:
Sorry, misunderstanding. Ignore above comment:)

Reply:
Just use proper formatting